my-app/src/reducers/news.ts

```diff
@@ -1,24 +1,30 @@
-import {FETCH_NEWS_START, FETCH_NEWS_SUCCESS, FETCH_NEWS_FAIL} from '../actions/newsAction'
+import { FETCH_NEWS_START, FETCH_NEWS_SUCCESS, FETCH_NEWS_FAIL } from '../const/const'
```

Comment:
why two const ?

Reply:
first constant is the folder, second is the file

Reply:
apiConstants ?